plugins/com.google.cloud.tools.eclipse.login/META-INF/MANIFEST.MF

```diff
@@ -15,32 +15,18 @@ Require-Bundle: com.google.gson;bundle-version="[2.7,3.0)",
  org.eclipse.ui,
```

Comment:
`jackson-core-asl`, `org.mortbay.jetty.server`, and `org.mortbay.jetty.util` don't seem necessary here, after your last commit. I had no problem logging in and out.